lockfile/Cargo.toml

```diff
@@ -18,6 +18,7 @@ log = "0.4.6"
 nom = "7.1.1"
 phylum_types = { git = "https://github.com/phylum-dev/phylum-types", branch = "development" }
 purl = "0.1.1"
+quick-xml = { version = "0.31.0", features = ["encoding", "serialize"] }
```

Comment:
Is there a good reason to use both serde-xml-rs and quick-xml? I'd prefer just using one of them.

Reply:
The `csharp` parser uses a method [non_contiguous_seq_elements](https://docs.rs/serde-xml-rs/0.6.0/serde_xml_rs/de/struct.Deserializer.html#method.non_contiguous_seq_elements) [here](https://github.com/phylum-dev/cli/blob/920aee4696d08b9819729ac88b1fe70e2131507a/lockfile/src/csharp.rs#L147) that's not available in `quick-xml`.

The tests pass if I remove it but it was explicitly added so I'm guessing it was needed at some point. I could look at writing a custom deserializer if we don't want to keep both crates.

Reply:
https://docs.rs/quick-xml/0.31.0/quick_xml/index.html#overlapped-lists

I think this handles the case I described. Will swap out the crate entirely.